package.json

```diff
@@ -4,6 +4,9 @@
   "description": "A terminal that tries to take care of you 💖",
   "main": "care.js",
   "author": "Monica Dinculescu <noms@google.com>",
+  "contributors": [
```

Comment:
I haven't been doing this for anyone else who has been sending PRs, so I'm inclined to just have the GitHub contributors reflect this. WDYT?